go-controller/pkg/util/net.go

```diff
@@ -150,3 +150,16 @@ func JoinIPNets(ipnets []*net.IPNet, sep string) string {
 	}
 	return b.String()
 }
+
+// JoinIPNetsIPOnly joins the string forms of an array of *net.IPNet,
+// as with strings.Join, but does not include the IP mask.
+func JoinIPNetsIPOnly(ipnets []*net.IPNet, sep string) string {
```

Comment:
maybe `JoinIPNetIPs`? It's a little shorter. But maybe it's less clear? Your call.